query-languages/m/m-spec-types.md

```diff
@@ -336,11 +336,21 @@ A value may be ascribed a type using the library function `Value.ReplaceType`. T
 
 When a value is ascribed a type, only a limited conformance check occurs:
 * The type being ascribed must be non-abstract, not nullable and compatible with the value's intrinsic (native) _primitive-type_.
-* When a custom type is being ascribed, it must match in structure with the value.
-  * For records: The type must be closed, must define the same number of fields as the value and must not contain any optional fields. (However, field names and field types are not checked.)
-  * For tables: The type must define the same number of columns as the value. (However, column names and column types are not checked.)
-  * For functions: The type must define the same number of required parameters, as well as the same number of optional parameters, as the value. (However, parameter names, parameter types and the function return type are not checked.)
-  * For lists: Unlike records, tables and functions, which can vary in their structure, a list's structure is always defined by a single item type, so a given list type is always structurally compatible with all other list types. (However, the item type is not checked during ascription.)
+* When a custom type that defines structure is ascribed, it must match the structure of the value.
+  * For records: The type must be closed, must define the same number of fields as the value and must not contain any optional fields. (However, field types are not checked.)
```

Comment:
Instead of a separate section, what about something like the following (based on what you had a few commits ago):

```suggestion
  * For records: The type must be closed, must define the same number of fields as the value, and must not contain any optional fields. (However, field names and field types are not checked and will override any existing values that differ.)
```

Reply:
(The same pattern would apply to the other types as well.)

Reply:
Revisions made. How do they look to you?

Reply:
Suggested some changes.

Reply:
Thanks! All changes committed.